server/src/testE2E/java/com/flipkart/varadhi/TopicTests.java

```diff
@@ -56,6 +56,8 @@ public void createTopic() {
                         ZNode.getResourceFQDN(topic.getProject(), topic.getName())
                 );
         makeCreateRequest(getTopicsUri(o1t1Project1), topic, 409, errorDuplicateTopic, true);
+        makeGetRequest(getTopicsUri(o1t1Project1)+"/"+topicName, TopicResource.class, 200);
+        makeDeleteRequest(getTopicsUri(o1t1Project1)+"/"+topicName, 200);
```

Comment:
lets add E2Es (1) multiple topics can be created across multiple projects for namespace creation (2) multiple tenants.